build-tools/reaper/build.gradle

```diff
@@ -1,3 +1,11 @@
+plugins {
+  id 'java'
+}
+
+
+targetCompatibility = '11'
```

Comment:
Shouldn't we also use the minimum runtime version here?

Reply:
true. removed this. minRuntime version is already set via allprojects in `build-tools`